src/functionalTest/resources/exception/all_fields_entered_uc.json

```diff
@@ -3,7 +3,7 @@
   "case_type_id": "case_type_id",
   "po_box": "po_box",
   "po_box_jurisdiction": "bulkscan",
-  "form_type": "SSCS1",
+  "form_type": "SSCS1PEU",
```

Comment:
Do you mind adding an equivalent test for the new SSCS1U form with the new fields?